modules/secure_storage_connector/variables.tf

```diff
@@ -1,20 +1,20 @@
-variable "resource_group_name" {
```

Comment:
the storage container will be provisioned in another tenant for dedicated (aka MI) customers. in order for them to create the federated credential file that will enable the dedicated instance to connect to it, they'll need to create their own resource group. so instead of passing in the name of a resource here, customers can provide a namespace to create their own resource group

TODO: think - what about self-managed?